salt/modules/selinux.py

```diff
@@ -164,7 +164,11 @@ def setenforce(mode):
             msg = 'Could not write SELinux enforce file: {0}'
             raise CommandExecutionError(msg.format(str(exc)))
 
-    config = '/etc/selinux/config'
+    if 'RedHat' in __grains__['os_family']:
```

Comment:
This should be `==` not an `in` comparison.

Reply:
Thx for the tip